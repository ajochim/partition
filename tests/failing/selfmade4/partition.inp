selfmade.gen
6
52
53
54
55
56
57
1.2
