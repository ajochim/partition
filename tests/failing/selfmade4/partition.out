8
9 9 6 7 6 6 4 4
1 9 34 35 36 38 39 50 51
2 7 8 10 32 33 37 48 49
3 6 11 31 46 47
4 5 12 13 14 44 45
15 16 21 22 30 40
17 18 19 23 29 41
20 24 28 42
25 26 27 43
